de.gonicus.gonnect.yml

```diff
@@ -0,0 +1,117 @@
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: gonnect
+
+modules:
+- name: kerberos
+  subdir: src
+  config-opts:
+    - --localstatedir=/var/lib
+    - --sbindir=${FLATPAK_DEST}/bin
+    - --disable-rpath
+    - --disable-static
+  cleanup:
+    - /bin
+    - /share/et
+    - /share/examples
+  post-install:
+    - install -Dm644 ../krb5.conf -t ${FLATPAK_DEST}/etc/
+  sources:
+    - type: archive
+      url: "https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz"
+      sha256: b7a4cd5ead67fb08b980b21abd150ff7217e85ea320c9ed0c6dadd304840ad35
+    - type: file
+      path: "krb5.conf"
+
+- name: pjsip
+  buildsystem: autotools
+  config-opts:
+  - --disable-video
+  - --enable-ext-sound
+  - CFLAGS=-fPIC -DPJ_HAS_IPV6=1
+  sources:
+  - type: git
+    url: https://github.com/pjsip/pjproject.git
+    tag: 2.15.1
+
+- name: openldap
+  config-opts:
+    - --disable-static
+    - --disable-slapd
+    - --disable-slurpd
+    - --disable-bdb
+    - --disable-hdb
+    - --with-tls=openssl
+  cleanup:
+    - /bin
+  sources:
+    - type: archive
+      url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.9.tgz"
+      sha256: 2cb7dc73e9c8340dff0d99357fbaa578abf30cc6619f0521972c555681e6b2ff
+
+- name: libusb
+  buildsystem: autotools
+  sources:
+  - type: git
+    url: https://github.com/libusb/libusb.git
+    tag: v1.0.27
+
+- name: hidapi
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://github.com/libusb/hidapi.git
+    tag: hidapi-0.14.0
+
+- name: gonnect
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_MODULE_PATH=/app/lib/x86_64-linux-gnu/cmake/Qt6
+  sources:
+  - type: git
+    url: https://github.com/gonicus/gonnect.git
+    tag: v1.0.0
+  - type: file
+    path: de.gonicus.gonnect.metainfo.xml
+  - type: file
+    path: de.gonicus.gonnect.releases.xml
+  - type: file
+    path: de.gonicus.gonnect.desktop
+  - type: file
+    path: de.gonicus.gonnect.krunner.desktop
+  - type: file
+    path: de.gonicus.gonnect.service
+  - type: file
+    path: de.gonicus.gonnect.svg
+  - type: file
+    path: de.gonicus.gonnect.search-provider.ini
+
+  post-install:
+    - install -Dm644 ${FLATPAK_ID}.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+    - install -Dm644 ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+    - install -Dm644 ${FLATPAK_ID}.krunner.desktop /app/share/krunner/dbusplugins/${FLATPAK_ID}.desktop
+    - install -Dm644 ${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    - install -Dm644 ${FLATPAK_ID}.releases.xml /app/share/metainfo/releases/${FLATPAK_ID}.releases.xml
+    - install -Dm644 ${FLATPAK_ID}.service /app/share/dbus-1/services/${FLATPAK_ID}.service
+    - install -Dm644 ${FLATPAK_ID}.search-provider.ini /app/share/gnome-shell/search-providers/${FLATPAK_ID}.search-provider.ini
+
+finish-args:
+  - --env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu;/usr/lib
```

Comment:
```suggestion
```

don't change the ld_library_path.

Reply:
KDE runtimes have qca installed in a non standard place. How to make it find `libqca-qt6.so.2` without adjusting the `LD_LIBRARY_PATH`?